### PR TITLE
rn_v94.md: add changes for v94.1

### PR DESCRIPTION
**Pull request by ricardosalveti**
# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Release note changes for the v94.1 stable release.

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by ricardosalveti** [comment]
@doanac @mike-sul @detsch anything worth saying about the ak-lite issues?

**Comment by doanac** [comment]
Docs for 61a0c20e320a92a2bda0398ceda23f10045f003f are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2736/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
lgtm

**Comment by mike-sul** [comment]
> @doanac @mike-sul @detsch anything worth saying about the ak-lite issues?

aklite/composectl fixes to check whether compose app images are installed and all its containers are running in the following cases:
1. One or more app images are hosted in the docker hub.
2. There are more than one version of the same app stored on a local host.
3. There are more than one app services/containers are based on the same container image.

**Review by angolini — APPROVED** [review]
I only ignored the vale errors. Other than that LGTM

File: release-notes/rn_v94.md [hunk 1]
```diff
@@ -39,6 +40,16 @@ TODO: Update with post-release findings
 
 ## General Updates
 
+### v94.1
+
```
Comment (angolini) [inline release-notes/rn_v94.md:44]: I would add here a line for the bugfix. I don't think we need another section, I think we can add a note only (but no strong opinion on this)

something like:
> - BUGFIX: it was detected that v94 included a regression on sound support for imx8mm-lpddr2-evk which is fixed by updating the linux kernel to a newer version https://github.com/foundriesio/meta-lmp/pull/1499.

**Review by angolini — APPROVED** [review]

File: release-notes/rn_v94.md (hunk 1 above)
Comment (ricardosalveti) [reply release-notes/rn_v94.md:49]: Added 'Alsa sound support fixed for imx8m targets using the linux-lmp-fslc kernel' as we don't yet have other similar bugfix entries with links.

**Comment by doanac** [comment]
Docs for 6ea481ea58ae2775e495e6720d57dc7aea2d3f1d are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2737/docs/artifacts/html/index.html

**Pull request merged**